otherlibs/threads/stdlib.ml

```diff
@@ -29,6 +27,7 @@ let () =
   register_named_value "Pervasives.array_bound_error"
     (Invalid_argument "index out of bounds")
 
+type nonrec exn = exn = ..
```

Comment:
Out of curiosity, does it affect client code in any way if reexport is done as above rather than just `type nonrec exn = exn`?

Reply:
Without the `= ..` it's not possible to extend `Stdlib.exn` — i.e. the following would be an error:

```ocaml
type Stdlib.exn += E
```

Reply:
Ok, thanks!  I was not aware that aliases were not expanded in this context.